package.json

```diff
@@ -19,7 +19,9 @@
     "coverage": "hardhat coverage",
     "test": "npm run test-incentives",
     "test-incentives": "TS_NODE_TRANSPILE_ONLY=1 hardhat test ./test/__setup.spec.ts ./test/AaveIncentivesController/*.spec.ts",
+    "test-incentives-upgrade-revision-2": "TS_NODE_TRANSPILE_ONLY=1 MAINNET_FORK=true hardhat test ./test-fork/incentives-revision-2-upgrade.spec.ts",
```

Comment:
The name of the file seems wrong.

Reply:
i think this was already addressed in the prev commit, but lmk if not the case